src/messages/Selection.hpp

```diff
@@ -38,7 +38,7 @@ struct SelectionItem {
 
     bool operator!=(const SelectionItem &b) const
     {
-        return this->operator==(b);
+        return !this->operator==(b);
     }
```

Comment:
As we're on C++ 20, we could remove the `operator!=` as it's synthesized, but not sure if all our compilers support this yet 🙃.

Reply:
by quick look 22 files define own `!=`, something nice to explore for sure

Reply:
I made this into an issue by itself https://github.com/Chatterino/chatterino2/issues/4895 with a new "Refactor" label - if you see any other small things like that feel free to make issues with that label